NOSQL/Assignment_doument_structure.txt

```diff
@@ -0,0 +1,37 @@
+Users
+{
+	id:1
+	user_name:"abc",
+	password:"zbxc123"
+	name:"anc asd",
+	date_of_birth:12-12-1990,
+	gender:"m or f"
+}
+
+Posts
+{
+	author_id:1
+	title:"post_title",
+	date_of_post:"11-2-2016 10:30",
```

Comment:
- Don't need to repeatedly specify "post", It could simply be posted_at, created_at.
- The value is a datetime not just date.